src/main/java/com/sarality/list/EditableListViewInitializer.java

```diff
@@ -0,0 +1,79 @@
+package com.sarality.list;
+
+import android.app.Activity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import com.sarality.datasource.DataSource;
+import com.sarality.form.FormBindings;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Initializer for an Editable List based on a LinearLayout that displays all rows
+ *
+ * @author satya@ (Satya Puniani)
+ */
+public class EditableListViewInitializer<T, H> {
+
+  private final Activity activity;
+  private final LinearLayout listView;
+  private final EditableListViewItemRenderer<T, H> renderer;
+  private List<FormBindings> bindingsList = new ArrayList<>();
+
+  private View emptyListView = null;
+
+  public EditableListViewInitializer(Activity activity, int listViewId, EditableListViewItemRenderer<T, H> renderer) {
```

Comment:
Can we add a TODO here to convert the LinearLayout into a ListView and manage the saving and loading of data from FormData once the view is recycled.